npre/umbral.py

```diff
@@ -12,7 +12,28 @@
 
 
 EncryptedKey = namedtuple('EncryptedKey', ['ekey', 're_id'])
-RekeyFrag = namedtuple('RekeyFrag', ['id', 'key'])
+
+
+class RekeyFrag(object):
+
+    DELIMETER = b"||"
+
+    def __init__(self, id, key):
+        self.id = id
+        self.key = key
+
+    def __bytes__(self):
+        return int(self.id).to_bytes(32, byteorder='big') + self.DELIMETER + int(self.key).to_bytes(32, byteorder='big')
```

Comment:
to_bytes is wrong here. I think, it's the other serialization function, from the ec library. It's not a bigint (although it might appear working)

Reply:
Gotcha. 

I'm looking at `serialize` and `deserialize`, and it's not immediately evident to me what they do:

```
def serialize(*args, **kwargs): # real signature unknown
    """ Serialize an element to a string """
    pass
```

Reply:
Yeah, just `serialize(self)` and a classmethod `deserialize(cls, ...)`?